cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -403,9 +403,10 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
     });
 
     after("Perform test data clean up", function () {
+        Archetype.open(true);
```

Comment:
Why do you need to open Archetype list explicitly? Archetype's delete method should call `open` by itself, no?

Reply:
As this is preventing afterAll failure if previous test is failing - let's leave it as is